src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/preview/main.js

```diff
@@ -270,6 +99,33 @@ define(['app-config'], function(AppConfig) {
                 }
             },
 
+            /**
+             * dom event wait for pause to redirect changes
+             * @param {Object} e
+             */
+            delayedUpdateEvent = function(e) {
```

Comment:
I would use underscore's debounce function instead of the next two methods: http://underscorejs.org/#debounce